### DISCUSSION TITLE
how to show horizontal mega menu on full screen

### DISCUSSION THREAD

**Discussion by neerajmalik** [saadeghi/daisyui#2671] [Q&A]
How to show horizontal responsive mega menu from left 0 and occupying the full screen rather aligned with parent menu option and taking only min-w-max?

**Comment by saadeghi** [comment]
Can you share an example link or a screenshot of a similar design?

**Reply by neerajmalik** [reply]
Please find enclosed screenshots.
From Left or Right Megamenu should open on full screen and also highlight current menu option in Navbar.  
Thanks,


![right_mega_menu_on_fullpage_issue](https://github.com/saadeghi/daisyui/assets/18030493/3d24a6ec-fc75-408c-b16f-319cdd1e4860)
![left_mega_menu_on_fullpage_issue](https://github.com/saadeghi/daisyui/assets/18030493/2bfa985a-485d-4603-bd9b-0f40165db45e)

Navbar -
      <div className="navbar bg-primary text-primary-content">
        <div className="navbar-start">
          <div className="hidden md:flex dropdown">
            <NonMobileMenu />
          </div>
        </div>
		
        <div className="navbar-end gap-3">
          <div className="hidden md:flex dropdown">
            <NonMobileMenu />
          </div>
        </div>
      </div>

const NonMobileMenu = () => {
    return (
        <>
            <ul tabIndex={0} className="menu menu-horizontal px-1">
                <li>
                    <details>
                        <summary>Mega Menu</summary>
                        <MegaMenuContent />
                    </details>
                </li>
            </ul>
        </>
    )
}

export default NonMobileMenu

const MegaMenuContent = () => {
    return (
        <>
            <ul tabIndex={0} className="menu sm:p-2 md:menu-horizontal md:min-w-max md:gap-20 rounded-box z-[1] dropdown-content">
                <li>
                    <a>Solutions</a>
                    <ul>
                        <li><a>Design</a></li>
                        <li><a>Development</a></li>
                        <li><a>Hosting</a></li>
                        <li><a>Domain register</a></li>
                    </ul>
                </li>
                <li>
                    <a>Enterprise</a>
                    <ul>
                        <li><a>CRM software</a></li>
                        <li><a>Marketing management</a></li>
                        <li><a>Security</a></li>
                        <li><a>Consulting</a></li>
                    </ul>
                </li>
                <li>
                    <a>Products</a>
                    <ul>
                        <li><a>UI Kit</a></li>
                        <li><a>Wordpress themes</a></li>
                        <li><a>Wordpress plugins</a></li>
                        <li>
                            <a>Open source</a>
                            <ul>
                                <li><a>Auth management system</a></li>
                                <li><a>VScode theme</a></li>
                                <li><a>Color picker app</a></li>
                            </ul>
                        </li>
                    </ul>
                </li>
                <li>
                    <a>Company</a>
                    <ul>
                        <li><a>About us</a></li>
                        <li><a>Contact us</a></li>
                        <li><a>Privacy policy</a></li>
                        <li><a>Press kit</a></li>
                    </ul>
                </li>
            </ul>
        </>
    )
}
